notifications-sw.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * UsersManager screenshot tests.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+var token;
+
+self.addEventListener('install', async function() {
+    self.token = new URL(location).searchParams.get('token');
+});
+
+self.addEventListener('activate', async function() {
```

Comment:
I'm not so much into service workers. The activate event might be only triggered once? Will it still work after closing and opening browser, after updating the service worker, and after updating to a new browser version and all these things? I reckon it looks good to use the activate event here like this just wanting to be sure.

I reckon the activate event will be executed again if there's an update. Do we need to do anything when the service worker?

> The activate event fires after a successful installation. You can use it to phase out older versions of the worker. We'll look at a basic example where we deleted stale cache entries.

I guess here's nothing to phase out of the older versions so should be all good I suppose.